data/presets/highway/speed_display.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "maki-square-stroked",
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "fields": [
+        "direction_point",
+        "direction_vertex"
```

Comment:
The wiki also suggests `maxspeed`; there’s an existing field we can use for that.

Reply:
> The wiki also suggests `maxspeed`; there’s an existing field we can use for that.

Decided to also add `maxspeed/advisory` for cases like this:
![image](https://user-images.githubusercontent.com/85302075/167562357-9ab8505e-f8cb-4703-a856-c39f961bf4b9.png)